scripts/aggregate_cells.py

```diff
@@ -0,0 +1,66 @@
+import os
```

Comment:
looks like you have a `aggregate_cells.py` and `_aggregate_cells.py`

Probably don't need both!

Reply:
👍